### PR TITLE
[Docs] How to specify specific GPU training and inference

### PR DESCRIPTION
**Pull request by yechenzhi**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation for this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/advanced_guides/how_to.md [hunk 1]
```diff
@@ -552,3 +552,24 @@ python ./tools/train.py \
 - `randomness.diff_rank_seed=True`，根据 rank 来设置不同的种子，`diff_rank_seed` 默认为 False。
 
 - `randomness.deterministic=True`，把 cuDNN 后端确定性选项设置为 True，即把`torch.backends.cudnn.deterministic` 设为 True，把 `torch.backends.cudnn.benchmark` 设为False。`deterministic` 默认为 False。更多细节见 https://pytorch.org/docs/stable/notes/randomness.html。
+
+## 指定特定GPU训练或推理
+
+如果你有多张GPU，比如8张，其编号分别为0, 1, 2, 3, 4, 5, 6, 7，使用单卡训练或推理时会默认使用卡0。如果想指定其他卡进行训练或推理，可以使用以下命令：
```
Comment (RangeKing) [inline docs/zh_cn/advanced_guides/how_to.md:558]:
```suggestion
如果你有多张 GPU，比如 8 张，其编号分别为 `0, 1, 2, 3, 4, 5, 6, 7`，使用单卡训练或推理时会默认使用卡 0。如果想指定其他卡进行训练或推理，可以使用以下命令：
```

File: docs/zh_cn/advanced_guides/how_to.md [hunk 2]
```diff
@@ -552,3 +552,24 @@ python ./tools/train.py \
...
+
+```shell
+CUDA_VISIBLE_DEVICES=5 python ./tools/train.py ${CONFIG}
+```
+
+如果设置`CUDA_VISIBLE_DEVICES`为-1或者一个大于GPU最大编号的数，比如8，将会使用CPU进行训练或者推理。
```
Comment (RangeKing) [inline docs/zh_cn/advanced_guides/how_to.md:564]:
```suggestion
如果设置 `CUDA_VISIBLE_DEVICES` 为 -1 或者一个大于 GPU 最大编号的数，比如 8，将会使用 CPU 进行训练或者推理。
```

File: docs/zh_cn/advanced_guides/how_to.md [hunk 3]
```diff
@@ -552,3 +552,24 @@ python ./tools/train.py \
...
+
+如果你想使用其中几张卡并行训练，可以使用如下命令：
+```shell
+CUDA_VISIBLE_DEVICES=0,1,2,3 ./tools/dist_train.sh ${CONFIG} ${GPU_NUM}
+```
+这里`GPU_NUM` 为4。另外如果在一台机器上多个任务同时多卡训练，需要设置不同的端口，比如以下命令：
```
Comment (RangeKing) [inline docs/zh_cn/advanced_guides/how_to.md:570]:
```suggestion
这里 `GPU_NUM`  为 4。另外如果在一台机器上多个任务同时多卡训练，需要设置不同的端口，比如以下命令：
```

File: docs/zh_cn/advanced_guides/how_to.md [hunk 4]
```diff
@@ -552,3 +552,24 @@ python ./tools/train.py \
 - `randomness.diff_rank_seed=True`，根据 rank 来设置不同的种子，`diff_rank_seed` 默认为 False。
 
 - `randomness.deterministic=True`，把 cuDNN 后端确定性选项设置为 True，即把`torch.backends.cudnn.deterministic` 设为 True，把 `torch.backends.cudnn.benchmark` 设为False。`deterministic` 默认为 False。更多细节见 https://pytorch.org/docs/stable/notes/randomness.html。
+
+## 指定特定GPU训练或推理
```
Comment (RangeKing) [inline docs/zh_cn/advanced_guides/how_to.md:556]:
```suggestion
## 指定特定 GPU 训练或推理
```

**Review by RangeKing — COMMENTED** [review]
Thank you for your kind contribution.
1. There should be a space between (English characters, numbers) and Chinese characters. I‘ve already modified in suggested change.
2. All the examples of commands are for training. It might be better to add a test example.
3. Please modify `en/advanced_guides/how_to.md` simultaneously.

**Comment by hhaAndroid** [comment]
@yechenzhi 中英文之间需要空格，数字和中文之间也需要空格，麻烦都处理下！

File: docs/zh_cn/advanced_guides/how_to.md [hunk 5]
```diff
@@ -552,3 +552,27 @@ python ./tools/train.py \
 - `randomness.diff_rank_seed=True`，根据 rank 来设置不同的种子，`diff_rank_seed` 默认为 False。
 
 - `randomness.deterministic=True`，把 cuDNN 后端确定性选项设置为 True，即把`torch.backends.cudnn.deterministic` 设为 True，把 `torch.backends.cudnn.benchmark` 设为False。`deterministic` 默认为 False。更多细节见 https://pytorch.org/docs/stable/notes/randomness.html。
+
+## 指定特定 GPU 训练或推理
+
+如果你有多张 GPU，比如 8 张，其编号分别为`0, 1, 2, 3, 4, 5, 6, 7`，使用单卡训练或推理时会默认使用卡 0。如果想指定其他卡进行训练或推理，可以使用以下命令：
```
Comment (RangeKing) [inline docs/zh_cn/advanced_guides/how_to.md:558]:
```suggestion
如果你有多张 GPU，比如 8 张，其编号分别为 `0, 1, 2, 3, 4, 5, 6, 7`，使用单卡训练或推理时会默认使用卡 0。如果想指定其他卡进行训练或推理，可以使用以下命令：
```

File: docs/zh_cn/advanced_guides/how_to.md [hunk 6]
```diff
@@ -552,3 +552,27 @@ python ./tools/train.py \
...
+
+```shell
+CUDA_VISIBLE_DEVICES=5 python ./tools/train.py ${CONFIG} #train
+CUDA_VISIBLE_DEVICES=5 python ./tools/test.py ${CONFIG} ${CHECKPOINT_FILE} #test
+```
+
+如果设置`CUDA_VISIBLE_DEVICES`为 -1 或者一个大于 GPU 最大编号的数，比如 8，将会使用 CPU 进行训练或者推理。
```
Comment (RangeKing) [inline docs/zh_cn/advanced_guides/how_to.md:565]:
```suggestion
如果设置 `CUDA_VISIBLE_DEVICES` 为 -1 或者一个大于 GPU 最大编号的数，比如 8，将会使用 CPU 进行训练或者推理。
```

File: docs/zh_cn/advanced_guides/how_to.md [hunk 7]
```diff
@@ -552,3 +552,27 @@ python ./tools/train.py \
...
+
+如果你想使用其中几张卡并行训练，可以使用如下命令：
+
+```shell
+CUDA_VISIBLE_DEVICES=0,1,2,3 ./tools/dist_train.sh ${CONFIG} ${GPU_NUM}
+```
+
+这里`GPU_NUM` 为 4。另外如果在一台机器上多个任务同时多卡训练，需要设置不同的端口，比如以下命令：
```
Comment (RangeKing) [inline docs/zh_cn/advanced_guides/how_to.md:573]:
```suggestion
这里 `GPU_NUM` 为 4。另外如果在一台机器上多个任务同时多卡训练，需要设置不同的端口，比如以下命令：
```

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**